packs/hello-universe-1.1.2/README.md

```diff
@@ -0,0 +1,51 @@
+# Hello Universe
+
+[Hello Universe](https://github.com/spectrocloud/hello-universe) is a demo web application utilized to help users learn more about [Palette](https://docs.spectrocloud.com/introduction) and its features. It functions as a standalone front-end application and provides users with a local click counter and funny Spectro Cloud-themed images.
+
+## Prerequisites
+
+- A Palette account.
+
+- A cluster profile where the Hello Universe pack can be integrated.
+
+- A Palette cluster with port `:8080` available. If port 8080 is not available, you can set a different port in the **values.yaml** file.
+
+- Ensure sufficient CPU resources within the cluster to allocate a minimum of 500 milliCPU and a maximum of 500 milliCPU per replica.
+
+## Parameters
+
+The following parameters are applied to the **hello-universe.yaml** manifest through the **values.yaml** file. Users do not need to take any additional actions regarding these parameters.
+
+| **Parameter**                     | **Description**                                                                | **Default Value**                           | **Required** |
+| --------------------------------- | ------------------------------------------------------------------------------ | ------------------------------------------- | ------------ |
+| `manifests.namespace`             | The namespace in which the application will be deployed.                       | `hello-universe`                            | Yes           |
+| `manifests.images.hello-universe` | The application image that will be utilized to create the containers.          | `ghcr.io/spectrocloud/hello-universe:1.1.2` | Yes           |
+| `manifests.apiEnabled`                  | The flag that indicates whether to deploy the UI application as standalone or together with the API server. | `false`                                      | Yes           |
```

Comment:
Can we add that an API service  and Postgres service will get deployed?

Reply:
Added this to the table now and further up in the intro